content/250-postgres/1100-integrations/500-vscode-agent.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'VS Code Agent' 
+description: 'Streamline Prisma ORM workflows, manage migrations, and create databases directly in Visual Studio Code using the Prisma Agent.' 
+metaTitle: 'Visual Studio Code Agent' 
+metaDescription: 'Manage Prisma ORM migrations and provision Prisma databases efficiently within Visual Studio Code using the Prisma Agent.' 
+tocDepth: 3 
+toc: true
+---
+
+## Overview
+
+[Visual Studio Code](https://code.visualstudio.com/docs/copilot/chat/chat-agent-mode) includes an **agent** mode (powered by GitHub Copilot) that automatically performs code changes and executes Prisma CLI commands based on your prompts.
+
+## Capabilities
+
+The [Prisma Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma) enables the agent to perform the following tasks using our Model Context Protocol (MCP) server:
+
+- Manage Prisma Data Platform projects.
+- Provision new databases.
+- Run migrations and generate the Prisma Client.
+- Generate Prisma schemas and client code.
+- Execute Prisma commands directly in the terminal.
```

Comment:
That has nothing to do with the agent mode extension. AIs can just generally call CLI comands.

Reply:
Here's what I'd write for this list:

- checking the status of your migrations (e.g. telling you if migrations haven't been applied)
- creating and running schema migrations for you
- authenticating you with the Prisma Console
- provisioning new Prisma Postgres instances so you can start coding right away